pareto/utilities/get_data.py

```diff
@@ -195,6 +370,9 @@ def get_data(fname, set_list, parameter_list, sum_repeated_indexes=False):
       The header should start in row 2, and the index column should start in cell A3.
       Column format: Does not require a header. Each set should be placed in one column,
       starting from column A and row 3. Data should be provided in the last column.
+    - set_list and parameter_list are optional parameters. When they are not given, tabs with
+      valid PARETO labels are read. Otherwise, only the specified tabs in set_list and
+      parameter_list are read.
```

Comment:
Correct me if I am wrong, but if there a use of reading this custom tabs as create model does not have this parameters and variables in them and thus would not be used to create and solve the model.

Reply:
@tarnold17  - good catch, this was how I initially implemented it, but I adjusted it after the first implementation. 
@JavalVyas2000 - The custom tabs are used in a few examples in the documentation (reading PARETO output files). It could also be used by developers who want to introduce a new parameter quickly for testing.